packages/procaptcha-react/src/components/CaptchaComponent.tsx

```diff
@@ -28,135 +28,143 @@ export interface CaptchaComponentProps {
+                        py={3}
                         sx={{
+                            // center the header
                             display: 'flex',
-                            flexDirection: 'column',
-                            // the min width of the popup before scroll bars appear
-                            minWidth: '300px',
+                            alignItems: 'center',
+                            width: '100%',
                         }}
+                        bgcolor={theme.palette.primary.main}
                     >
-                        <Box
-                            px={2}
-                            py={3}
+                        <Typography
                             sx={{
-                                // center the header
-                                display: 'flex',
-                                alignItems: 'center',
-                                width: '100%',
+                                color: '#ffffff',
+                                fontWeight: 700,
                             }}
-                            bgcolor={theme.palette.primary.main}
                         >
-                            <Typography
-                                sx={{
-                                    color: '#ffffff',
-                                    fontWeight: 700,
-                                }}
-                            >
-                                {t('WIDGET.SELECT_ALL')}
-                                {': '}
-                            </Typography>
-                            <Typography
-                                px={1}
-                                sx={{
-                                    color: '#ffffff',
-                                    fontWeight: 700,
-                                    textTransform: 'uppercase',
-                                    fontSize: theme.typography.h6.fontSize,
-                                }}
-                            >
-                                {`${at(props.challenge.captchas, props.index).captcha.target}`}
-                            </Typography>
-                        </Box>
-
-                        <Box {...addDataAttr({ dev: { cy: 'captcha-' + props.index } })}>
-                            <CaptchaWidget challenge={captcha} solution={solution} onClick={onClick} />
-                        </Box>
-                        <Box
-                            px={2}
-                            py={1}
+                            {t('WIDGET.SELECT_ALL')}
+                            {': '}
+                        </Typography>
+                        <Typography
+                            px={1}
                             sx={{
-                                display: 'flex',
-                                alignItems: 'center',
-                                justifyContent: 'center',
-                                width: '100%',
+                                color: '#ffffff',
```

Comment:
should this come from the theme?

Reply:
No, this is the topbar for the expanded captcha which is blue background white text